app/install/dcos.md

```diff
@@ -68,58 +68,237 @@ have basic knowledge of [DC/OS]({{ page.links.dcos }}),
+    - `dbname`: This parameter configures the name of the kong database.
+    - `persistence`: This parameter enables persistent volumes for postgresql.
+
+    Install postgresql using `postgres.json` file from the repo:
 
     ```bash
-    $ dcos marathon app add postgres.json
+    $ dcos package install postgresql --options=postgres.json
     ```
 
 5. **Deploy Kong**
 
-    Using the `kong_<postgres|cassandra>.json` file from the kong-dist-dcos
-    repo, deploy Kong in the cluster. Template registers Kong's `proxy` and
-    `admin` ports on Marathon-LB as service ports `10001` and `10002`
-    respectively:
+    Now we have an external load balancer and Kong supported datastore
+    running. Using the `kong` package from Universe repo, deploy Kong
+    with following option:
+
+    ```json
+    {
+      "service": {
+        "name": "kong",
+        "instances": 1,
+        "cpus": 1,
+        "mem": 512,
+        "role": "*"
+      },
+      "configurations": {
+        "log-level": "notice",
+        "database": {
+          "migrations": true,
+          "use-cassandra": false
+        },
+        "postgres": {
+          "host": "postgresql.marathon.l4lb.thisdcos.directory",
+          "port": 5432,
+          "database": "kong",
+          "user": "kong",
+          "password": "kong"
+        },
+        "cassandra": {
+          "contact-points": "node-0.cassandra.mesos, node-1.cassandra.mesos, node-2.cassandra.mesos",
+          "port": 9042,
+          "keyspace": "kong"
+        }
+      },
+      "networking": {
+        "proxy": {
+          "external-access": true,
+          "vip-port": 8000,
+          "vip-port-ssl": 8443,
+          "virtual-host": "<vhost>",
+          "https-redirect": true,
+          "service-port": 10201
+        },
+        "admin": {
+          "external-access": true,
+          "vip-port": 8001,
+          "vip-port-ssl": 8444,
+          "https-redirect": false,
+          "service-port": 10202
+        }
+      }
+    }
+    ```
+
+    It configures Kong as follows:
+
+    | Config  |     Description |
+    |----------|:---------------|
+    | `configurations.log_level`|Sets the Kong's log_level configuration.|
+    | `configurations.custom-envs`|A space-separated list of Kong configurations.|
+    | `configurations.database.use-cassandra`|If `true`, Cassandra is used as the Kong database.|
+    | `configurations.database.migration`| If `true`, Kong will run migrations during start.|
+    | `configurations.postgres.host`| PostgreSQL host name.|
+    | `configurations.postgres.port`| PostgreSQL port.|
+    | `configurations.postgres.database`| PostgreSQL database name.|
+    | `configurations.postgres.user`| PostgreSQL username.|
+    | `configurations.postgres.password`| PostgreSQL password.|
+    | `configurations.cassandra.contact-points`| Comma separated list of Cassandra contact points.|
+    | `configurations.cassandra.port`| Port on which Cassandra listening for query.|
```

Comment:
"Port on which Cassandra is listening for queries" or simply "Port on which Cassandra is listening"